Async/ResolveCache.php

```diff
@@ -32,9 +32,7 @@ class ResolveCache implements \JsonSerializable
     private $force;
 
     /**
-     * @param string        $path
```

Comment:
so the idea here is to only document parameters that are not fully documented via the method signature?

Reply:
This come from the `@Symfony` ruleset.

I could set `no_superfluous_phpdoc_tags` to `false`, revert changes and stick with the GitHub Actions part.

Then we could discuss of the rules to apply to this project.